--- a/Makefile.obexd	2024-02-11 08:42:22.628633669 -0600
+++ b/Makefile.obexd	2024-02-11 08:43:28.843393253 -0600
@@ -8,10 +8,10 @@
 
 uninstall-hook:
 	rm -f $(DESTDIR)$(SYSTEMD_USERUNITDIR)/dbus-org.bluez.obex.service
+endif
 
 dbussessionbusdir = $(DBUS_SESSIONBUSDIR)
 dbussessionbus_DATA = obexd/src/org.bluez.obex.service
-endif
 
 EXTRA_DIST += obexd/src/org.bluez.obex.service
 
--- a/obexd/src/org.bluez.obex.service	2024-02-11 05:58:02.219018824 -0600
+++ b/obexd/src/org.bluez.obex.service	2024-02-11 06:01:36.493206927 -0600
@@ -1,4 +1,4 @@
 [D-BUS Service]
 Name=org.bluez.obex
-Exec=/bin/false
+Exec=/usr/libexec/bluetooth/obexd
 SystemdService=dbus-org.bluez.obex.service
