--- /usr/portage/x11-libs/gtk+/gtk+-3.22.19.ebuild	2018-08-30 18:40:02.000000000 -0500
+++ ./gtk+-3.22.19.ebuild	2018-10-21 14:50:04.027722162 -0500
@@ -11,7 +11,7 @@
 
 LICENSE="LGPL-2+"
 SLOT="3"
-IUSE="aqua broadway cloudprint colord cups examples +introspection test vim-syntax wayland +X xinerama"
+IUSE="aqua atk-bridge broadway cloudprint colord cups examples +introspection test vim-syntax wayland +X xinerama"
 REQUIRED_USE="
 	|| ( aqua wayland X )
 	xinerama? ( X )
@@ -48,7 +48,7 @@
 		>=x11-libs/libxkbcommon-0.2[${MULTILIB_USEDEP}]
 	)
 	X? (
-		>=app-accessibility/at-spi2-atk-2.5.3[${MULTILIB_USEDEP}]
+		atk-bridge? ( >=app-accessibility/at-spi2-atk-2.5.3[${MULTILIB_USEDEP}] )
 		x11-libs/libX11[${MULTILIB_USEDEP}]
 		>=x11-libs/libXi-1.3[${MULTILIB_USEDEP}]
 		x11-libs/libXext[${MULTILIB_USEDEP}]
@@ -120,6 +120,11 @@
 	# gtk-update-icon-cache is installed by dev-util/gtk-update-icon-cache
 	eapply "${FILESDIR}"/${PN}-3.22.2-update-icon-cache.patch
 
+    # get rid of gtk3-atk-bridge crap
+	if ! use atk-bridge; then
+		eapply "${FILESDIR}"/${PN}-3.22.19.atk-bridge.patch
+	fi
+
 	eautoreconf
 	gnome2_src_prepare
 }
@@ -143,6 +148,7 @@
 		$(use_enable X xkb) \
 		$(use_enable X xrandr) \
 		$(use_enable xinerama) \
+		$(use_with atk-bridge) \
 		--disable-papi \
 		--disable-mir-backend \
 		--enable-man \
