diff -ur ./distutils-r1.eclass /tmp/distutils-r1.eclass
--- ./distutils-r1.eclass	2021-09-26 05:16:17.841325425 -0500
+++ /tmp/distutils-r1.eclass	2021-09-26 06:25:29.939135524 -0500
@@ -741,8 +741,10 @@
 			local basename=${f##*/}
 
 			debug-print "${FUNCNAME}: installing wrapper at ${bindir}/${basename}"
-			_python_ln_rel "${path}${EPREFIX}"/usr/lib/python-exec/python-exec2 \
-				"${path}${bindir}/${basename}" || die
+			local dosym=dosym
+			[[ ${EAPI} == [67] ]] && dosym=dosym8
+			"${dosym}" -r /usr/lib/python-exec/python-exec2 \
+				"${bindir#${EPREFIX}}/${basename}"
 		done
 
 		for f in "${non_python_files[@]}"; do
diff -ur ./python-r1.eclass /tmp/python-r1.eclass
--- ./python-r1.eclass	2021-09-26 05:26:29.642575998 -0500
+++ /tmp/python-r1.eclass	2021-09-26 06:26:35.297736717 -0500
@@ -865,7 +865,9 @@
 	# install the wrappers
 	local f
 	for f; do
-		_python_ln_rel "${ED%/}/usr/lib/python-exec/python-exec2" "${f}" || die
+		local dosym=dosym
+		[[ ${EAPI} == [67] ]] && dosym=dosym8
+		"${dosym}" -r /usr/lib/python-exec/python-exec2 "${f#${ED}}"
 	done
 }
 
diff -ur ./python-utils-r1.eclass /tmp/python-utils-r1.eclass
--- ./python-utils-r1.eclass	2021-09-26 05:26:49.562453496 -0500
+++ /tmp/python-utils-r1.eclass	2021-09-26 06:27:45.177315673 -0500
@@ -712,8 +712,9 @@
 	)
 
 	# install the wrapper
-	_python_ln_rel "${ED%/}"/usr/lib/python-exec/python-exec2 \
-		"${ED%/}/${wrapd}/${newfn}" || die
+	local dosym=dosym
+	[[ ${EAPI} == [67] ]] && dosym=dosym8
+	"${dosym}" -r /usr/lib/python-exec/python-exec2 "${wrapd}/${newfn}"
 
 	# don't use this at home, just call python_doscript() instead
 	if [[ ${_PYTHON_REWRITE_SHEBANG} ]]; then
