--- frontend/audio.c.orig	2010-10-24 15:36:19.673485313 +0800
+++ frontend/audio.c	2010-10-24 15:36:43.323775214 +0800
@@ -347,7 +347,7 @@
 {
     int ret;
     unsigned int i;
-    long *sample_buffer24 = (long*)sample_buffer;
+    int *sample_buffer24 = (int*)sample_buffer;
     char *data = malloc(samples*aufile->bits_per_sample*sizeof(char)/8);
 
     aufile->total_samples += samples;
@@ -391,7 +391,7 @@
 {
     int ret;
     unsigned int i;
-    long *sample_buffer32 = (long*)sample_buffer;
+    int *sample_buffer32 = (int*)sample_buffer;
     char *data = malloc(samples*aufile->bits_per_sample*sizeof(char)/8);
 
     aufile->total_samples += samples;

