--- ./configure.ac.lib64	2006-01-13 14:53:23.000000000 +0100
+++ ./configure.ac	2006-08-15 21:19:10.398503342 +0200
@@ -135,7 +135,7 @@
 			yes|no)
 				;;
 			*)
-				if ((test -f $withval/lib/libupsclient.a) &&
+				if (
 					(test -f $withval/include/upsclient.h) &&
 					(test -f $withval/include/parseconf.h))
 				then
@@ -212,7 +212,7 @@
 	dnl 	NUT_HEADER="`pkg-config --cflags libupsclient`"
 	dnl else
 	dnl pkg-config --exists libupsclient --silence-errors
-	if (test -f /usr/lib/libupsclient.a)
+	if (test -f /usr/lib/libupsclient.a || test -f /usr/lib64/libupsclient.a)
 	then
 		AC_MSG_RESULT(found (libupsclient.a))
 	 	NUT_LIB="-L/usr/lib -lupsclient"
